### DISCUSSION TITLE
Using Behaviors in Avalonia and WPF article

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#7261] [General]
Published an article on the Behavior Pattern and its use in Avalonia and WPF: [Behavior Pattern - Visual Behaviors in WPF and Avalonia with Samples](https://www.codeproject.com/Articles/5320737/Behavior-Pattern-Visual-Behaviors-in-WPF-and-Avalo)

**Comment by maxkatz6** [comment]
@npolyak this link is wrong

**Reply by npolyak** [reply]
I fixed it. Thanks Max!
